R/item_deletion_h.R

```diff
@@ -184,7 +184,8 @@ item_deletion_h <- function(propsel = NULL,
 
   # Call PartInv with the full item set under strict invariance
   store_str[[1]] <- 
-    PartInv(propsel = propsel, cut_z = cut_z, weights_item, weights_latent,
+    PartInv(cfa_output = NULL, propsel = propsel, cut_z = cut_z, weights_item, 
```

Comment:
It seems `cfa_output` is not an argument for `PartInv()` in this version. Does this cause any issues?